src/main/webapp/script.js

```diff
@@ -31,11 +33,19 @@ const gJSONtasks = JSON.stringify(gTasks);
 const gDefaultTask = {taskID: 0, projectID: 0, name: 'Default Task',
   description: 'Default task description...', status: 'none',
   users: [], subtasks: []};
-
 // TODO: fill gUsers and gJSONusers and gDefaultUser (similar to above).
-const gUsers = {};
-const gJSONusers = {};
-const gDefaultUser = {userID: 0, name: 'Default Username'}; // Add attributes
+const gUsers =
+  {user1: {userID: 1, name: 'User 1', skills: 'Art, Writing',
+    major: 'Studio Art', numTaskCompleted: 5, admin: true},
+  user2: {userID: 2, name: 'User 2', skills: 'Object Oriented Programming',
+    major: 'Computer Science', numTaskCompleted: 8, admin: false},
+  user3: {userID: 3, name: 'User 1', skills: 'Leadership, Organization',
+    major: 'Chemistry', numTaskCompleted: 4, admin: false}};
+
+const gJSONusers = JSON.stringify(gUsers);
+const gDefaultUser = {userID: 0, name: 'Default Username',
+  skills: 'Default Skills', major: 'Default Major',
+  numTaskCompleted: 'Default Number', admin: false}; // Add attributes
```

Comment:
I would put multiple skills as a list of strings, like `skills: ['Art', 'Writing']`. Same with major if double majoring will be considered.